submissions/2024-09-11-OpenFF-Sulfur-Optimization-Training-Coverage-Supplement-v1.0/README.md

```diff
@@ -0,0 +1,61 @@
+# OpenFF Sulfur Optimization Training Coverage Supplement v1.0
+
+## Description
+
+An optimization data set created to improve the training coverage of sulfonic
+and phosphonic acids, sulfone, sulfonate, sulfinyl, sulfoximine, sulfonamides,
+thioether, and 1,3-thiazole groups. The molecules in this data set were selected
+from the ChEMBL 34 database.
```

Comment:
Are these full molecules from the CHEMBL database or just fragments? If they're fragments, could you please briefly describe the fragmentation and selection procedure? (e.g. on rotatable bonds, RECAP -- if they were clustered and selected for diversity, then which metric was used, if size was a consideration, etc).

The link below to a commit version of curato is fantastic (it fixes a lot of woes that happen when people point to a branch they later update!), but it's not clear from just the repo itself what was done to select molecules.

Reply:
These are actually full molecules even though they're pretty small. I clustered them in a couple of cases but mostly just sorted by size and manually scrolled through the first ~500 or so molecules picking out ones that caught my eye. That sounded pretty arbitrary to me, so I didn't think to add more to the description, but I'm happy to do so now if that sounds reasonable. Maybe something like

The molecules in this data set were manually selected from a subset of the smallest matching structures in the ChEMBL 34 database.

Reply:
Yup, that's a lot clearer, thanks!

Reply:
I'll merge to get these going over the weekend, but in the future it'd be more user-friendly to have this in the dataset description as well (i.e. re-generate the dataset).